perception/autoware_crosswalk_traffic_light_estimator/README.md

```diff
@@ -2,33 +2,39 @@
 | :---------------------------- | :------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :------------ |
 | `use_last_detect_color`       | `bool`   | If this parameter is `true`, this module estimates pedestrian's traffic signal as RED not only when vehicle's traffic signal is detected as GREEN/AMBER but also when detection results change GREEN/AMBER to UNKNOWN. (If detection results change RED or AMBER to UNKNOWN, this module estimates pedestrian's traffic signal as UNKNOWN.) If this parameter is `false`, this module use only latest detection results for estimation. (Only when the detection result is GREEN/AMBER, this module estimates pedestrian's traffic signal as RED.) | `true`        |
-| `last_detect_color_hold_time` | `double` | The time threshold to hold for last detect color.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  | `2.0`         |
+| `last_detect_color_hold_time` | `double` | The time threshold to hold for last detected vehicle traffic light color.                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | `2.0`         |
+| `last_colors_hold_time`       | `double` | The time threshold to hold for history detected pedestrian traffic light color.                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
```

Comment:
Could you please add the units?

Reply:
It's added: https://github.com/autowarefoundation/autoware.universe/pull/6724/commits/e529247b56d761aa95e1c08828fc975a03d2d57b#diff-5a2a13b8e47e19b81499e006c7fdbf8993e9d94ecbc5d78f03ffed824274533cR28